src/architecture/_GeneralModuleFiles/swig_conly_data.i

```diff
@@ -102,7 +100,7 @@ ARRAYASLIST(unsigned int)
     for (i = 0; i < PySequence_Length($input); i++) {
         PyObject *o = PySequence_GetItem($input,i);
```

Comment:
I think this object is leaking.

Reply:
I've made several comments like these. Reference counting is tricky when using the Python C API, so feel free to correct me if you don't think these are actually leaking.

Reply:
I believe you're right. It seems like `PySequence_GetItem` is indeed a strong reference instead of borrowed reference so needs to be DECREF'd.